lib/ddtrace/contrib/grape/configuration/settings.rb

```diff
@@ -24,6 +24,7 @@ class Settings < Contrib::Configuration::Settings
           end
 
           option :service_name, default: Ext::SERVICE_NAME
+          option :dont_report_4xx, default: false
```

Comment:
As discussed, let's rename this to `error_statuses` and have it default to nil and accept an array of error status ranges, such as: `['400-428, '430-599']`